src/page/AddressStep.page.ts

```diff
@@ -0,0 +1,11 @@
+import { $, ElementFinder, promise } from 'protractor';
+
+export class AddressStepPage {
+  private get proceedToCheckoutButton(): ElementFinder {
+    return $('#center_column > form > p > button > span');

```

Comment:
why not declare this in a variable instead of a method?

Reply:
because this way is specified on the workshop